src/stac_api_validator/validations.py

```diff
@@ -514,6 +525,9 @@ def validate_api(
     if auth_query_parameter and (xs := auth_query_parameter.split("=", 1)):
         r_session.params = {xs[0]: xs[1]}
 
+    if auth_headers:
+        r_session.headers.update(ast.literal_eval(auth_headers))
+
```

Comment:
`auth_headers` is a misleading name for this argument, since it's updating the `headers`. I recommend changing the argument name to `headers`.